problems/5.loops.js

```diff
@@ -2,7 +2,11 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
+function evenNumbers(num) {
+  if(Array.length % 2 === 0){
```

Comment:
We don't have an array here. We need to make an array and iterate from 0 to num and populate the array with the even elements.